.circleci/config.yml

```diff
@@ -73,6 +72,18 @@ aliases:
       - unit_tests_node8
 
 commands:
+  notify-status:
+    parameters:
+      condition:
+        type: boolean
+        default: false
+    steps:
+      - when:
+          condition: << parameters.condition >>
+          steps:
+            - slack/status:
+              channel: eng-react-integration-status
```

Comment:
Maybe tag in the core team for failure?

Reply:
@wardpeet this will run every night. and 90% of the time it should pass. So I don't really want to spam a common channel that will create too much noise. We can always change later if we want/need

Reply:
@sidharthachatterjee i tried to figure out tagging and it's not working. I'm just going to leave it for now and we'll see if we need tagging later